### ISSUE TITLE
`instanceof` doesn't minify correctly

### ISSUE THREAD

**Issue by meezwhite** [putoutjs/minify#4]
The following doesn't seem to minify correctly:

```js
class Animal {
  constructor(name) {
    this.name = name;
  }
}

const lion = new Animal('lion');
const isLion = lion instanceof Animal; // <-- this minifies incorrectly
```

The last line minifies to something like:

```js
// ...
const b = new Animal('lion');
const a = binstanceofAnimal; // <-- missing spaces
```

**Comment by coderaiser** [comment]
Thank you! Fixed with https://github.com/putoutjs/printer/commit/2ee5b24b0890bb4a53b90a956ff5e49336216bfe. Landed in [`@putout/minify@1.17.3`](https://github.com/putoutjs/minify/releases/tag/v1.17.3) 🎉.

Is it works for you?

**Comment by meezwhite** [comment]
It works, thank you!
